.doc_gen/metadata/aurora_metadata.yaml

```diff
@@ -1079,9 +1205,42 @@ aurora_Scenario_GetStartedClusters:
               snippet_tags:
                 - cpp.example_code.aurora.client_configuration
                 - cpp.example_code.aurora.get_started_clusters
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rust_dev_preview/examples/aurora
+          excerpts:
+            - description: A library containing the scenario-specific functions for the Aurora Scenario.
+              snippet_files:
+                - rust_dev_preview/examples/aurora/src/aurora_scenario/mod.rs
+            - description: Tests for the library using automocks around the RDS Client wrapper.
+              snippet_files:
+                - rust_dev_preview/examples/aurora/src/aurora_scenario/tests.rs
+            - description: A binary to run the scenario from front to end, using inquirer to allow the user to make some decisions.
```

Comment:
```suggestion
            - description: A binary to run the scenario from front to end, using inquirer so that the user can make some decisions.
```